### PR TITLE
Delete

### PR DESCRIPTION
**Pull request by ViktoriyaSynytsya**

### CODE REVIEW

File: page_objects/menu_page.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+let BasePage = require("../base/base.page");
+let Button = require("../elements/button.element")
+
+let womenTabLocator = by.xpath('//a[.="Women"]')
+let productLocator = by.xpath('(//a[.="Summer Dresses"])[1]')
+let addtocart1Locator = by.xpath('(//span[contains(text(),"Add to cart")])[1]')
+let continueBtnLocator = by.css('.continue span')
+let addtocart2Locator = by.xpath('(//span[contains(text(),"Add to cart")])[2]')
+let addtocart3Locator = by.xpath('(//span[contains(text(),"Add to cart")])[3]')
```
Comment (nbabii) [inline page_objects/menu_page.js]: Please use selectors without `text()` usage

**Review by nbabii — CHANGES_REQUESTED** [review]

File: page_objects/menu_page.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+let BasePage = require("../base/base.page");
+let Button = require("../elements/button.element")
+
+let womenTabLocator = by.xpath('//a[.="Women"]')
+let productLocator = by.xpath('(//a[.="Summer Dresses"])[1]')
+let addtocart1Locator = by.xpath('(//span[contains(text(),"Add to cart")])[1]')
```
Comment (nbabii) [inline page_objects/menu_page.js]: `addtocart1Locator, addtocart2Locator, addtocart3Locator` - use single locator and element object

**Review by nbabii — CHANGES_REQUESTED** [review]

File: page_objects/menu_page.js (hunk 1 above)
Comment (ViktoriyaSynytsya) [reply page_objects/menu_page.js]: Fixed

File: page_objects/menu_page.js (hunk 2 above)
Comment (ViktoriyaSynytsya) [reply page_objects/menu_page.js]: Fixed

File: page_objects/menu_page.js [hunk 3]
```diff
@@ -36,18 +36,18 @@ class MenuPage extends BasePage {
...
-        return  new Button(element(addtocart2Locator), 'Summer Dress2');
-    };
-    getAddProduct3Element() {
-        return  new Button(element(addtocart3Locator), 'Summer Dress3');
+    getAddProductElement(index) {
+        let finalLocator =  Object.assign({}, addtocartLocator);
+        console.log('test'+ JSON.stringify(finalLocator));
+        finalLocator.value = finalLocator.value.replace("INDEX", index);
+        console.log('test'+ JSON.stringify(finalLocator));
+        console.log('test2'+ JSON.stringify(addtocartLocator));
```
Comment (nbabii) [inline page_objects/menu_page.js]: Please remove all console logs, they were just for debugging needs

**Review by nbabii — CHANGES_REQUESTED** [review]

File: page_objects/cart_page.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
+const { browser } = require("protractor");
```
Comment (nbabii) [inline page_objects/cart_page.js]: please remove

File: page_objects/menu_page_delete.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+    getWomenBtnElement() {
+        return  new Button(element(womenTabLocator), 'Women Tab');
+    };
+    getProductElement() {
+        return  new Button(element(productLocator), 'Summer Dress Tab');
+    };
+    getAddProductElement(index) {
+        let finalLocator =  Object.assign({}, addtocartLocator);
+        console.log('test'+ JSON.stringify(finalLocator));
```
Comment (nbabii) [inline page_objects/menu_page_delete.js]: Please remove

File: page_objects/menu_page_delete.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        finalLocator.value = finalLocator.value.replace("INDEX", index);
+        console.log('test'+ JSON.stringify(finalLocator));
```
Comment (nbabii) [inline page_objects/menu_page_delete.js]: remove

File: page_objects/menu_page_delete.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        console.log('test2'+ JSON.stringify(addtocartLocator));
```
Comment (nbabii) [inline page_objects/menu_page_delete.js]: remove

File: specs/delete_spec.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
+let MenuPage = require ("../page_objects/menu_page")
```
Comment (nbabii) [inline specs/delete_spec.js]: user lowercase as it is variable `menuPage `

File: specs/delete_spec.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+let indexPage = require ("../page_objects/index_page")
+let CartPage = require ("../page_objects/cart_page")
```
Comment (nbabii) [inline specs/delete_spec.js]: `cartPage `

File: specs/delete_spec.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+let MainPage = require ("../page_objects/main_page");
```
Comment (nbabii) [inline specs/delete_spec.js]: `mainPage `

**Review by nbabii — CHANGES_REQUESTED** [review]

File: specs/delete_spec.js (hunk 10 above)
Comment (ViktoriyaSynytsya) [reply specs/delete_spec.js]: fixed

File: specs/delete_spec.js (hunk 9 above)
Comment (ViktoriyaSynytsya) [reply specs/delete_spec.js]: fixed

File: specs/delete_spec.js (hunk 8 above)
Comment (ViktoriyaSynytsya) [reply specs/delete_spec.js]: fixed

File: page_objects/menu_page_delete.js (hunk 7 above)
Comment (ViktoriyaSynytsya) [reply page_objects/menu_page_delete.js]: fixed

File: page_objects/menu_page_delete.js (hunk 6 above)
Comment (ViktoriyaSynytsya) [reply page_objects/menu_page_delete.js]: fixed

File: page_objects/menu_page_delete.js (hunk 5 above)
Comment (ViktoriyaSynytsya) [reply page_objects/menu_page_delete.js]: fixed

File: page_objects/cart_page.js (hunk 4 above)
Comment (ViktoriyaSynytsya) [reply page_objects/cart_page.js]: fixed

File: page_objects/menu_page.js (hunk 3 above)
Comment (ViktoriyaSynytsya) [reply page_objects/menu_page.js]: fixed

**Pull request merged**